ProjectSettings/EditorBuildSettings.asset

```diff
@@ -4,4 +4,10 @@
 EditorBuildSettings:
   m_ObjectHideFlags: 0
   serializedVersion: 2
-  m_Scenes: []
\ No newline at end of file
+  m_Scenes:
+  - enabled: 1
+    path: Assets/HoloToolkit/Utilities/Scenes/HeadsetAdjustment.unity
```

Comment:
Please revert :)

Reply:
This was intentional.

Reply:
Why?

Reply:
Why not?

Reply:
If I'm working in the Toolkit, why do I need HeadsetAdjustment? Why do I need SceneLauncher? Saying "Why not?" doesn't answer any of those questions. I build my open scene, and that's all I care about. If I want to use these, let me, but don't force me to. It'll add more work for me to uncheck/delete them and then make sure not to commit that change.

Reply:
If you import the toolkit into your own project they will not be included.

Testing this project's scenes on device, on the other hand, might be pretty handy to have this enabled.

Reply:
@ForrestTrepte even suggested putting in _all_ the scenes. I just choose the two that are included in the main toolkit package.

Reply:
So, SceneLauncher becomes useful if I'm testing more than one scene, or do I get anything with just one? I'm not 100% clear on why it's useful, which is why I'm asking about it becoming the default.

>Yeah I just wanted to know your reasoning why you wouldn't want it, is all.

And I just wanted to know your reasoning why you did :)

Reply:
> So, SceneLauncher becomes useful if I'm testing more than one scene, or do I get anything with just one?

I'd like to think so.  That's mainly the reason why I was going to add it.  To test the example and test scenes out easier.

Reply:
Also, does this change cause me to do any additional work? Do I need to manually add my scene somewhere to show up in the launcher, or can I just build with my scene and it works?

Reply:
I was even half tempted to move the SceneLauncher scene out of the main toolkit and put it in an example folder then reference _all_ the scenes in the toolkit.

Reply:
>Also, does this change cause me to do any additional work? Do I need to manually add my scene somewhere to show up in the launcher, or can I just build with my scene and it works? 

It would depend on if we all all the toolkit scenes.

- If we have all the toolkit scenes in the build settings:
    - If you're working on a scene that's already in the toolkit, no extra work. Deploy and test to your hearts content.
    - If you're working on a scene that's not in the toolkit, and will be added to it, then you should add it to the build settings, the update the SceneLauncher.prefab with the enabled button.

- If we don't include all the toolkit scenes:
    - You will need to add it to the build settings, the update the SceneLauncher.prefab with the enabled button.
    - Or alternatively delete the two scenes and add your own.

Reply:
The original reason that I created the Scene Launcher was to make it easier to test all the HoloToolkit scenes on device. From that standpoint, I like @StephenHodgson 's idea of moving it to examples, including all of the HoloToolkit scenes, and automatically enabling the new scene button. (We might also have to do something to tickle SceneLauncher.prefab to update itself. My original implementation automatically updated the list of scenes when hitting Play or making a build but I think this may no longer be the case with this change.) The result of including all scenes would be that when people run the HoloToolkit by default they have the capability to try out and test all the things that HoloToolkit does.

I do, however, also recognize that something would be lost with this approach. I think @keveleigh may be pointing out that there's an advantage to having no scenes in the build settings. In this case, it seems like Unity picks whatever scene you have open in the editor and automatically includes that one in the build. The result is that you automatically build and run whatever scene you currently have open in the editor, which is more convenient when you're working on a single scene at a time. Also, I suspect that if we include all scenes the build will be slower.

If my understanding is correct, we have a choice between:
1) Making the default (most convenient) behavior to run the scene you currently have open, but requiring someone to go to extra effort to run and test all the HoloToolkit scenes.
- or -
2) Making the default (most convenient) behavior to run all the scenes, but requiring someone to go to extra effort to directly run their current scene.

I'm not sure what the best choice is. Is there a way we could make both behaviors very convenient?

Reply:
Perhaps for now the most expedient approach would be to revert the EditorBuildSettings change and complete this pull request, and then to open a separate issue about changing to include all scenes.

Reply:
> I'm not sure what the best choice is. Is there a way we could make both behaviors very convenient?

Yeah probably.  But I'll do it later in a separate PR.

I agree. Just revert for now and open a new issue for it.